temoa-uiuc/Snakefile

```diff
@@ -21,9 +21,9 @@ rule all:
         expand(r"data_files/{db}_{scenario}_model/{scenario}_OutputLog.log",
                zip,
                db=DBS, scenario=SCENARIOS),
-        # expand(r"data_files/{db}_{scenario}_model/{scenario}.lp",
-        #        zip,
-        #        db=DBS, scenario=SCENARIOS),
+        expand(r"data_files/{db}_{scenario}_model/{scenario}.lp",
```

Comment:
Tell snakemake to take in .lp file files?

Reply:
Temoa builds .lp files. Snakemake executes a temoa run and knows its finished by the presence of a .lp file in the correct directory.